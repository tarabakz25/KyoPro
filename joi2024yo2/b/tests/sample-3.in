5 5 3
50 2
70 4
20 5
30 1
10 3
4 2 4
5 1 5
2 3 4
