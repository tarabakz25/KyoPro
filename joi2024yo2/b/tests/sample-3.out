85
170
50
